UI/ObjectListWnd.cpp

```diff
@@ -2399,7 +2401,10 @@ void ObjectListWnd::ObjectRightClicked(GG::ListBox::iterator it, const GG::Pt& p
     if (popup.Run()) {
         switch (popup.MenuID()) {
         case 1: {
-            ObjectDumpSignal(object_id);
+            if (GetOptionsDB().Get<bool>("UI.objects-hide-dump"))
+                ErrorLogger() << "Bug: dump contxt-menu clicked, when it should not be possible";
+            else
```

Comment:
These lines seem pretty pointless to me.   I don't even see why you would have needed them as part of your debugging since the behavior announces itself as it is tested.